main.py

```diff
@@ -14,59 +15,56 @@
     parser.add_argument("--no-bio", help="Don't fetch biography", action="store_true")
     parser.add_argument("--no-album", help="Don't download pictures", action="store_true")
     parser.add_argument("--no-video", help="Don't download videos", action="store_true")
-    parser.add_argument("--log", help="Print log messages", action="store_true")
+    parser.add_argument("--log", help="Set log level (error|warning|info|debug)")
     args = parser.parse_args()
 
-    if args.log:
-        write_log = True
+    if args.log is not None:
+        numeric_level = getattr(logging, args.log.upper(), None)
+        if not isinstance(numeric_level, int):
+            raise ValueError(f"Invalid log level: {args.log}")
+        logging.basicConfig(level=numeric_level)
 
     if args.target:
-        if not args.no_bio:
-            write_bio = True
-        if not args.no_album:
-            write_pictures = True
-        if not args.no_video:
-            write_videos = True
+        write_bio = not args.no_bio
+        write_pictures = not args.no_album
+        write_videos = not args.no_video
 
         if args.site == "freeones":
             # Scrape freeones.com, write logs
-            f = FreeOnes(write_log)
+            f = FreeOnes(write_bio, write_pictures, write_videos)
 
             # Select babe
             f.select_target(args.target)
 
             # Biography
-            print("Biography")
-            f.write_bio(write_bio)
+
             bio = f.bio()
 
             # Photo albums
-            print("Albums")
-            f.write_albums(write_pictures)
+            logging.info("Albums")
             while (a := f.next_album()) is not None:
-                print(a.meta())
+                logging.debug(a.meta())
                 ps = 0
                 while (p := a.next_photo(download=True)) is not None:
```

Comment:
this logic needs to use the `--no-album` flag.

it actually looks like the album/video logic needs to be split into 3 flags:
```
--no-album : do not consider photo albums at all
--no-video : do not consider videos at all
--no-download : if albums or videos are considered, only return their data but do not download the image/video files
```